### PR TITLE
Integrate popup

### PR DESCRIPTION
**Pull request by benedikt-richter**
The ReplaceAttributionPopup is integrated to enable the full workflow.
Furthermore, a integration test is added and the definition for two attributions to be equal is improved (we now ignore the confidence for that decision).

### CODE REVIEW

File: src/Frontend/Components/ReplaceAttributionPopup/__tests__/ReplaceAttributionPopup.test.tsx [hunk 1]
```diff
@@ -31,8 +38,9 @@ describe('ReplaceAttributionPopup and do not change view', () => {
...
     setupTestState(store);
 
-    expect(screen.queryByText('Warning')).toBeTruthy();
-    expect(screen.queryAllByText('React').length).toBe(2);
+    expect(screen.queryByText('Replacing an attribution')).toBeTruthy();
+    expect(screen.queryByText('React')).toBeTruthy();
+    expect(screen.queryByText('Vue')).toBeTruthy();
 
     fireEvent.click(screen.queryByText(ButtonTitle.Cancel) as Element);
     expect(getOpenPopup(store.getState())).toBe(null);
```
Comment (nicarl) [inline src/Frontend/Components/ReplaceAttributionPopup/__tests__/ReplaceAttributionPopup.test.tsx:46]: Could we also add a test here for clicking okay?

File: src/Frontend/state/helpers/save-action-helpers.ts [hunk 2]
```diff
@@ -457,7 +457,17 @@ export function attributionForTemporaryPackageInfoExists(
 ): boolean {
   const manualAttributions = getManualAttributions(state);
   return Object.values(manualAttributions).some((packageInfo) =>
-    isEqual(packageInfo, packageInfoToMatch)
+    attributionsAreEqual(packageInfo, packageInfoToMatch)
+  );
+}
+
+export function attributionsAreEqual(
```
Comment (nicarl) [inline src/Frontend/state/helpers/save-action-helpers.ts:464]: For next time: here a commit message with additional information would be super valuable for the future

**Review by nicarl — APPROVED** [review]
Great integration test :+1: Works nicely

File: src/Frontend/Components/ReplaceAttributionPopup/__tests__/ReplaceAttributionPopup.test.tsx (hunk 1 above)
Comment (nicarl) [reply src/Frontend/Components/ReplaceAttributionPopup/__tests__/ReplaceAttributionPopup.test.tsx:46]: Somehow optional as tested by integration test

**Review by leslielazzarino — APPROVED** [review]

**Pull request merged**